GitExtUtils/GitUI/TaskManager.cs

```diff
@@ -108,8 +135,24 @@ public void JoinPendingOperations()
         /// The readability of the callstack is improved by calling <see cref="ExceptionExtensions.Demystify"/>.
         internal async Task ReportExceptionOnMainThreadAsync(Exception ex)
         {
-            await JoinableTaskFactory.SwitchToMainThreadAsync();
-            Application.OnThreadException(ex.Demystify());
+            try
+            {
+                await JoinableTaskFactory.SwitchToMainThreadAsync();
+                Application.OnThreadException(ex.Demystify());
+            }
+            catch (Exception exceptionWhileReporting)
+            {
+                try
+                {
+                    await Console.Error.WriteLineAsync(exceptionWhileReporting.ToString());
+                    await Console.Error.WriteLineAsync(ex.ToString());
+                    await Console.Error.WriteLineAsync(ex.StackTrace);
```

Comment:
Write the exception to `Trace` stream so that we can observe it even outside a debugger?

Reply:
done